Quick/Reporting/XCTestDriver.h

```diff
@@ -0,0 +1,18 @@
+//
+//  XCTestDriver.h
+//  Quick
+//
+//  Created by Paul Young on 1/3/15.
+//  Copyright (c) 2015 Brian Ivan Gesiak. All rights reserved.
+//
```

Comment:
nitpick: Remove these comments from all newly added files--the convention in Quick is to not include any template comments at the top of a file.